packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx

```diff
@@ -25,9 +24,7 @@ export const MuiThemeDecorator: Decorator = (Story, context) => {
       <StorybookThemeProvider theme={odysseyTheme}>
         <CssBaseline />
         <div style={styles} lang={locale}>
-          <ScopedCssBaseline>
-            <Story />
-          </ScopedCssBaseline>
```

Comment:
Talked with @bryancunningham-okta , this MUI imported `ScopedCssBaseline` is bringing in unwanted styles for Storybook. Not sure if we want to create a separate PR to remove it or if removing it here is fine